### ISSUE TITLE
hass.helpers.service will stop working

### ISSUE THREAD

**Issue by Clappsepp** [dolezsa/thermal_comfort#395]
2024-09-05 20:27:11.486 WARNING (MainThread) [homeassistant.helpers.frame] Detected that custom integration 'thermal_comfort' accesses hass.helpers.service. This is deprecated and will stop working in Home Assistant 2024.11, it should be updated to import functions used from service directly at custom_components/thermal_comfort/__init__.py, line 162: hass.helpers.service.async_register_admin_service(, please create a bug report at https://github.com/dolezsa/thermal_comfort/issues

**Comment by igorsantos07** [comment]
Looks like a dupe of #370, #374 and #376, which is said to be fixed at #377. A search for the error message is useful before creating any issue.

**Comment by MaxiKingXXL** [comment]
I have the same error in my log. The issue is not solved in v2.2.2 master branch yet.


Logger: homeassistant.helpers.frame
Quelle: helpers/frame.py:184
Erstmals aufgetreten: 22:33:19 (2 Vorkommnisse)
Zuletzt protokolliert: 22:33:21

Detected that custom integration 'thermal_comfort' accesses hass.helpers.service. This is deprecated and will stop working in Home Assistant 2024.11, it should be updated to import functions used from service directly at custom_components/thermal_comfort/__init__.py, line 162: hass.helpers.service.async_register_admin_service(, please create a bug report at https://github.com/dolezsa/thermal_comfort/issues

**Comment by MaxiKingXXL** [comment]
Look like the issue is solved in master but the version was not updated.
I have done a manual reload which solve the issue for me now.
Would be good if someone could update to 2.2.3

**Comment by mzaufl** [comment]
I added the Code Snipplet (fixed at #377) to "custom_components/thermal_comfort/__init__.py" and now it works again. Warning is disappeared.

**Comment by pejotigrek** [comment]
> I added the Code Snipplet (fixed at #377) to "custom_components/thermal_comfort/**init**.py" and now it works again. Warning is disappeared.

can you please elaborate on that?
I am on v2.2.2, I've compared my installed `__init__.py` with the one in the repository, they look the same but still, I've got the aboe warning in the logs..

**Comment by sven-debug** [comment]
I can add, the problem still exists and is not solved in 2.2.2. Would be a pity if the integration would stop working because it is a great one!

**Comment by Maarc** [comment]
Hi! I have the same issue with the latest version.

```
Detected that custom integration 'thermal_comfort' accesses hass.helpers.service. This is deprecated and will stop working in Home Assistant 2024.11
```

It looks like the fix #377 has been merged on the master branch on August 11th. However, no new release has been published since January 8th (version 2.2.2).

Is there any chance someone could bump up a new release to get this one fixed before 2024.11? :-)
